portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsTextField.testcase

```diff
@@ -505,14 +502,43 @@ definition {
 	}
 
 	@description = "Verify that a Predefined Value can be set"
-	@ignore = "Test Stub"
 	@priority = "5"
 	test PredefinedValueCanBeAdded {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-131341 PredefinedValueCanBeAdded pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.addTextField(fieldPositionNumber = "0");
+
+		Form.editTextAdvancedTab(key_fieldPredefinedText = "Test Text");
 
+		AssertElementPresent(
+			key_fieldName = "Predefined Value",
+			key_fieldValue = "Test Text",
+			locator1 = "FormViewBuilder#PREDEFINED_VALUE_AT_FORM");
+
+		Form.save();
+
+		Form.publishForm();
+
+		FormsAdminNavigator.gotoPublishedForm();
+
+		Refresh();
```

Comment:
I just ran the test again and `Refresh();` is not necessary. `FormsAdminNavigator.gotoPublishedForm()` is necessary though, because the ticket asks to open the published form.
I'll make the necessary changes! Thanks!